tsl/test/sql/CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ set(SOLO_TESTS
   bgw_reorder_drop_chunks
   chunk_api
   compress_bgw_reorder_drop_chunks
+  compression_ddl
```

Comment:
These tests have been added here but not removed from TEST_FILES_DEBUG/TEST_TEMPLATES? Is that intended?

Reply:
These do different things. One keeps track of the files, the other one decides which tests are solo tests. If you look at, for example, `chunk_api` and `continuous_aggs_bgw` they are in both locations.